$NetBSD: patch-configure.ac,v 1.2 2019/08/23 01:48:00 schmonz Exp $

Remove ncursesw hack which actually breaks macOS.
Use portable shell string-equality test.

--- configure.ac.orig	2018-11-23 02:26:09.000000000 +0000
+++ configure.ac
@@ -90,11 +90,6 @@ AC_ARG_ENABLE(utf8, [  --enable-utf8   E
 
 if test "$utf8" = "yes"; then
   libncursesw=ncursesw
-  # Simply called libncurses on OS X
-  case "$host_os" in
-    *darwin*) libncursesw=ncurses
-    ;;
-  esac
 
   AC_CHECK_LIB([$libncursesw], [mvaddwstr], [],
       [AC_MSG_ERROR([*** Missing development libraries for ncursesw])])
@@ -212,9 +207,9 @@ fi
 
 HAS_SEDTR=no
 AC_CHECK_PROG([SED_CHECK],[sed],[yes],[no])
-if test x"$SED_CHECK" == x"yes" ; then
+if test x"$SED_CHECK" = x"yes" ; then
   AC_CHECK_PROG([TR_CHECK],[tr],[yes],[no])
-  if test x"$TR_CHECK" == x"yes" ; then
+  if test x"$TR_CHECK" = x"yes" ; then
     HAS_SEDTR=yes
   fi
 fi
